docs/topics/modules/using_code_assistant.adoc

```diff
@@ -2,15 +2,17 @@
 = Using the code assistant
 
 . Return to the Che workspace browser tab. If you closed your earlier workspace tab, use the following instructions:
-.. In {osio}, click the btn:[Create] tab.
-.. The default `Codebases` view lists your project workspace:
+
+.. In {osio}, click the *Create* tab. The default *Codebases* view lists your project workspace:
 +
 image::existing_workspace.png[Existing Workspace]
 +
 .. Click btn:[Open] to view the existing workspace in a new tab. If the new tab does not load, ensure your browser is not blocking pop-up windows.
-.. When the workspace loads, navigate to the following file: `src` > `main` > `Java` > `io.openshift.booster` > `HttpApplication.java`.
+. When the workspace loads, navigate to the following file: `src` > `main` > `Java` > `io.openshift.booster` > `HttpApplication.java`.
 +
 image::project_dir.png[Project Directory]
++
+
 . In the `HttpApplication.java` file, add a line break at line 9 after the imports section.
```

Comment:
Is it really 'imports' section? I.e. isn't it capitalized in the file?

Also, it's in a file, so put backticks around the word that appears in the file